nmdc_runtime/site/ops.py

```diff
@@ -724,6 +735,33 @@ def export_json_to_drs(
     return ["/objects/" + drs_object["id"]]
 
 
+@op(
+    description="NCBI Submission XML file rendered in a Dagster Asset",
+    out=Out(description="XML content rendered through Dagit UI"),
+)
+def ncbi_submission_xml_asset(context: OpExecutionContext, data: str):
+    filename = "ncbi_submission.xml"
+    file_path = os.path.join(context.instance.storage_directory(), filename)
```

Comment:
I haven't seen this `storage_directory` function before. Is there any kind of documentation on it? I can't seem to find any.

Reply:
Good catch again!

So some context as to what I'm trying to do/experiment with here. A hard requirement that came from @aclum was that we should be able to download the content, in this case, XML content that is rendered through the Dagit interface as a file on our local system.

This `storage_directory()` is part of https://docs.dagster.io/deployment/dagster-instance#local-artifact-storage which we can use to store files. Then those files need to be allowed/able to be "served up" through Dagit somehow. I'm still trying to iron out the kinks with respect to that detail, but do let me know if you have any ideas.